images/build/cross/Makefile

```diff
@@ -28,7 +28,7 @@ IMGNAME = kube-cross
 # - v1.100.0-go1.17-bullseye.0 satisfies SemVer regex, while:
```

Comment:
can we remove these arches from `linux/ppc64le linux/s390x` lines from this arch or we can also feed the platforms variable to the variants.yaml to build only on the supported platform.

for testing I have removed these problematic platforms and build went through here - https://github.com/kubernetes/release/pull/3782

Reply:
thanks, I can do that. 

shall we wait for [this proposal](https://groups.google.com/a/kubernetes.io/g/dev/c/LBlLCWdFN-o/m/YtGXW-qZBAAJ) to be commonly agreed on?

Reply:
anyway, carried your commit, I guess that PR approve will have to take consideration of that proposal approve.